x-pack/plugins/cases/server/client/cases/bulk_get.ts

```diff
@@ -74,33 +71,29 @@ export const bulkGet = async <Field extends keyof Case = keyof Case>(
         operation: Operations.bulkGetCases,
       });
 
-    const requestForTotals = ['totalComment', 'totalAlerts'].some(
-      (totalKey) => !fields || fields.includes(totalKey)
-    );
-
-    const commentTotals = requestForTotals
-      ? await attachmentService.getter.getCaseCommentStats({
-          caseIds: authorizedCases.map((theCase) => theCase.id),
-        })
-      : new Map();
+    const commentTotals = await attachmentService.getter.getCaseCommentStats({
+      caseIds: authorizedCases.map((theCase) => theCase.id),
+    });
 
     const flattenedCases = authorizedCases.map((theCase) => {
-      const { alerts, userComments } = commentTotals.get(theCase.id) ?? {
+      const { userComments } = commentTotals.get(theCase.id) ?? {
         alerts: 0,
         userComments: 0,
       };
 
       const flattenedCase = flattenCaseSavedObject({
         savedObject: theCase,
         totalComment: userComments,
-        totalAlerts: alerts,
       });
 
       if (!fields?.length) {
```

Comment:
Do we still need this? `fields` is always defined.

Reply:
No, you are right.